config/initializers/devise.rb

```diff
@@ -239,7 +239,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
Why did this change?

Reply:
Main application uses this option. Added for compatibility.